From 30a42caf02dc91a00667b3d548395af79389db3a Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Fri, 22 Mar 2024 13:09:39 +0100
Subject: .chj-home/init 000: [PATCH 171/191] init: import gpg keys first

Before checkout runs. Huh. Not sure why it ever worked, because
mod-user does it, too?
---
 .chj-home/init | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/.chj-home/init b/.chj-home/init
--- a/.chj-home/init
+++ b/.chj-home/init
@@ -115,6 +115,9 @@ EOF
 set +eu
 set -x
 
+gpg --import /opt/chj/chjize/cj-key.asc || true
+gpg --import /opt/chj/chjize/cj-key-2.asc || true
+
 /opt/chj/chjize/bin/chj-checkout _ignored_ https://github.com/pflanze/chj-emacs .chj-emacs '^v(\d+)$'
 
 ln -s .chj-emacs/.emacs
